### ISSUE TITLE
Missing Code in peer_sts.py file

### ISSUE THREAD

**Issue by mzfr** [P2PSP/simulator#43]
In peer_sts.py file [line 19](https://github.com/P2PSP/simulator/blob/master/src/core/peer_sts.py#L19) there is an incomplete for loop.

**Comment by vicente-gonzalez-ruiz** [comment]
Yes, this code is unfinished. It should implement [this](https://p2psp.github.io/#x1-20001) functionality.
